litgpt/deploy/serve.py

```diff
@@ -160,7 +160,8 @@ def setup(self, device: str):
             config = json.load(fp)
             chat_template = config.get("chat_template", None)
             if chat_template is None:
-                raise ValueError("chat_template not found in tokenizer config file.")
+                print("The tokenizer config does not contain chat_template, falling back to a default.")
```

Comment:
not sure if `print` is the right one here

Reply:
What would you suggest?  We use print all over this file